src/fides/api/ctl/sql_models.py

```diff
@@ -287,7 +287,10 @@ class System(Base, FidesBase):
     ingress = Column(JSON)
 
     users = relationship(
-        "FidesUser", secondary="systemmanager", back_populates="systems"
+        "FidesUser",
+        secondary="systemmanager",
+        back_populates="systems",
+        lazy="selectin",
```

Comment:
I ran into issues with this line in `export.py` 
```python
system_users = ", ".join([user.username for user in system.get("users", [])])
```

I needed to add this line to make resolving the linked users work in an async route






https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#select-in-loading